# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome2/gnome-vfs/pc-requires-gobject.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./gnome-vfs-2.0.pc.in.orig	2004-12-05 00:06:32.683417144 +0100
+++ ./gnome-vfs-2.0.pc.in	2004-12-05 00:06:40.165279728 +0100
@@ -7,6 +7,6 @@
 Name: gnome-vfs
 Description: The GNOME virtual file-system libraries
 Version: @VERSION@
-Requires: bonobo-activation-2.0 libbonobo-2.0 gconf-2.0 gthread-2.0 gmodule-2.0
+Requires: bonobo-activation-2.0 libbonobo-2.0 gconf-2.0 gthread-2.0 gmodule-2.0 gobject-2.0
 Libs: -L${libdir} -lgnomevfs-2
 Cflags: -I${includedir}/gnome-vfs-2.0 -I${libdir}/gnome-vfs-2.0/include
